blog/2023/02/the-injectivity-radii-of-the-unitary-groups.md

```diff
@@ -0,0 +1,206 @@
++++
+title = "The injectivity radii of the unitary groups"
+tags = ["linear algebra", "group theory", "unitary", "rotations", "special orthogonal", "special unitary"]
+date = Date(2023, 02, 22)
+description = "Working out the injectivity radius for the unitary, orthogonal, special unitary, and special orthogonal groups."
+published = date
+rss_pubdate = date
+rss_description = description
+maxtoclevel = 2
++++
+
+\tableofcontents
+
+## Introduction
+
+Suppose you held up an object and began rotating it at a fixed speed.
+After some amount of time, you stop rotating it.
+If I know the starting pose of the object and the amount of time that has passed, under what conditions can I also tell you the rotational velocity (both direction and speed) of the spinning object?
+
+There's really just one condition: that the rotational distance (i.e. angle) between the initial and final positions does not exceed some maximum value.
```

Comment:
to be precise: one condition to be able to answer the question before with a yes ;) But maybe that is clear from context.

Reply:
The question before is not a yes or no question though.

Reply:
Ah I did not read that one careful enough it asks for a condition. Then it's fine.